build-prep.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/sh
+
 set -o errexit
 
 . ./build.properties
 
-JRE_FILENAME=`echo "${JRE_URL##*/}"`
+# If you want the current release
+
+if [ "${USE_MVN}" = 'true' ]; then
```

Comment:
Just a note that using our internal Maven repo (or any private location) prevents us from using Docker Hub to build images, and any public location would be a legal no no.

Reply:
_(BTW, that comment on line 7 needs deleting)_

@rgauss - if we set `USE_MVN` to false, it would build *the latest java release* (only) on docker hub, but I fear I'm slipping into some multiple-branch hell. Thoughts? I've less experience in a multi-headed hydra of a CI solution.

Reply:
I'm not sure there's much more we can do until Oracle fixes things.